utils/realtek-poe/Makefile

```diff
@@ -27,9 +27,10 @@ define Package/realtek-poe/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_DIR) $(1)/etc/config
 	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_DIR) $(1)/etc/uci-defaults
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/realtek-poe $(1)/usr/bin/
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/etc/init.d/poe $(1)/etc/init.d/
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/files/etc/config/poe $(1)/etc/config/
+	$(CP) ./files/etc/uci-defaults/30-poe $(1)/etc/uci-defaults/30-poe
```

Comment:
Shouldn't this use some sort of `INSTALL_` macro, so that the file is installed with the correct permissions?

Reply:
I pulled the behavior from `libreswan`, but yeah, there are certainly newer changesets showing the use of the `INSTALL_BIN` macro -- e.g. commit 3219c50828914d82da375e959b06da3f6f40d2a4.

Adjusting.

Reply:
Fixed with the push of bb0fb2b7f864a5c0a9e0cf9ffcb05d82e200c21b.